Slim/Plugin/ACLFiletest/strings.txt

```diff
@@ -1,7 +1,9 @@
 PLUGIN_ACL_FILETEST
 	EN	Enable ACL filetests
 	NL	Schakel ACL-bestandtest in
+	NO	Aktiver ACL-filtest
 	
 PLUGIN_ACL_FILETEST_DESC
 	EN	This plugin allows you to check file permissions based on ACLs on some systems. Do not enable this plugin unless you know your system does support ACLs and Perl's 'use filetest'.
-	NL	Deze plug-in maakt het op sommige systemen mogelijk om bestandsrechten op basis van ACL's te controleren. Schakel deze plug-in alleen in als je zeker weet dat je systeem ACL's and Perl's 'use filetest' ondersteunt.
\ No newline at end of file
+	NL	Deze plug-in maakt het op sommige systemen mogelijk om bestandsrechten op basis van ACL's te controleren. Schakel deze plug-in alleen in als je zeker weet dat je systeem ACL's and Perl's 'use filetest' ondersteunt.
+	NO	Denne plugin-modulen lar deg kontrollere filtilganger basert p� ACL'er p� noen systemer. Ikke aktiver denne modulen med mindre du vet at systemet ditt st�tter ACL'er og Perls 'use filetest'.
```

Comment:
Ah, that's the penalty for being lazy and using Notepad.

I'll fire up Emacs and fix the encoding tomorrow (hopefully).


2016-08-30 20:23 GMT+02:00 michaelherger <notifications@github.com>:

> In Slim/Plugin/ACLFiletest/strings.txt
> <https://github.com/Logitech/slimserver/pull/110#discussion_r76851326>:
>
> > \ No newline at end of file
> > +	NL	Deze plug-in maakt het op sommige systemen mogelijk om bestandsrechten op basis van ACL's te controleren. Schakel deze plug-in alleen in als je zeker weet dat je systeem ACL's and Perl's 'use filetest' ondersteunt.
> > +	NO	Denne plugin-modulen lar deg kontrollere filtilganger basert p� ACL'er p� noen systemer. Ikke aktiver denne modulen med mindre du vet at systemet ditt st�tter ACL'er og Perls 'use filetest'.
>
> Looks like the encoding of this file isn't correct. Make sure your text
> editor can handle UTF-8 (and does so).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Logitech/slimserver/pull/110/files/3b71770242024bf1d6354065b1cec89c832b781c#r76851326>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQYZHheanNn0zCZEjysIK_9lbA62W-PEks5qlHU0gaJpZM4JwxYj>
> .
>